src/main/java/properties/EnviConfigStorage.java

```diff
@@ -0,0 +1,40 @@
+package properties;
+
+/***
+ * Project environment data
+ */
+public class EnviConfigStorage {
```

Comment:
I would not say that this class is storage as you do not store data here
It's more like config or reader or smth like that.

Also don't use shortcuts(Envi) in names. try to use full words

Reply:
fixed